src/components/navbar/index.tsx

```diff
@@ -131,6 +131,7 @@ const NavBar: React.FC<NavBarProps> = ({ tokens }) => {
           if (asyncRequestIsComplete(tokens)) {
             dispatch(logout());
             history.push(Routes.HOME);
+            history.go(0);
```

Comment:
❓ What is this doing? I'm not super familiar with how the history object but I read [https://reactrouter.com/web/api/history](url) and it seems like `history.go(0)` would move the history object back 0 times.

Reply:
`history.go(0)` esentially just lets reloads the page

Reply:
oh interesting this is super useful to know